mariadb/content.md

```diff
@@ -190,3 +190,150 @@ For restoring data. You can use the `docker exec` command with the `-i` flag, si
 ```console
 $ docker exec -i some-%%REPO%% sh -c 'exec mysql -uroot -p"$MARIADB_ROOT_PASSWORD"' < /some/path/on/your/host/all-databases.sql
 ```
+
+## Creating backups with Mariabackup
+
+To perform a backup using Mariabackup, an additional volume for the backup needs to be included when the container is started like this:
+
+```console
+$ docker run --name some-%%REPO%% -v /my/own/datadir:/var/lib/mysql -v /my/own/backupdir:/backup -e MARIADB_ROOT_PASSWORD=my-secret-pw -d %%IMAGE%%:latest
+```
+
+Mariabackup will run as the `mysql` user in the container, so the permissions on `/backup` will need to ensure that it can be written to by this user:
+
+```console
+$ docker exec some-%%REPO%% chown mysql: /backup
+```
+
+To perform the backup:
+
+```console
+$ docker exec some-%%REPO%% gosu mysql mariabackup --backup --target-dir=/backup --user=root --password=my-secret-pw
```

Comment:
I'd recommend just using `--user mysql` for all of these examples, rather then possibly confusing users with `gosu`. It works for both `docker run` and `docker exec`.

Reply:
Much nicer. Thanks.